integration_tests/e2e/prisonerIncentiveLevelDetails.cy.ts

```diff
@@ -0,0 +1,169 @@
+import moment from 'moment'
+import Page from '../pages/page'
+import PrisonerIncentiveLevelDetailsPage from '../pages/prisonerIncentiveLevels/prisonerIncentiveLevelDetailsPage'
+
+const iepSummaryResponse = {
```

Comment:
similarly, might be easier to reuse [emptyIncentiveSummaryForBooking](https://github.com/ministryofjustice/hmpps-incentives-ui/blob/main/server/testData/incentivesApi.ts#L127)